infrastructure/ansible/roles/traffic_ops/templates/cdn.conf.j2

```diff
@@ -50,6 +50,7 @@
       "address" : "{{ to_smtp_address }}"
    },
    "use_ims": {{ to_use_ims | bool | lower }},
+   "role_based_permissions": {{{ to_use_rbp | bool | lower }},
```

Comment:
Upon closer review, I believe there is one too many `{` here. I believe it will either render a single `{`, which is not what we want, or Ansible may fail to process this template.